gui/services/migrations/0025_snmp_zilstat.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.8 on 2019-04-22 04:56
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import freenasUI.freeadmin.models.fields
+
+
+class Migration(migrations.Migration):
```

Comment:
I think we should be smarter about this field when upgrading.

Defaulting it to true if the service is marked to start on boot.